ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx

```diff
@@ -459,6 +462,10 @@ export function CreateSiteDialogContainer(props: CreateSiteDialogContainerProps)
     setDialog({ inProgress: false });
   }
 
+  useMount(() => {
+    hasGlobalPermission('list_plugins').subscribe(setHasListPluginPermission);
+  });
```

Comment:
Can we move this check to the effect that fetches the marketplace bps? That way we also remove the `hasListPluginPermission` dependency from it.